CMakeLists.txt

```diff
@@ -438,7 +437,9 @@ if(UNIX AND APPLE)
             VERBATIM
         )
         add_custom_target(compile_assets ALL DEPENDS "${GENERATED_ASSETS_CAR}")
-        install(FILES "${GENERATED_ASSETS_CAR}" DESTINATION "${RESOURCES_DEST_DIR}")
+        if(EXISTS "${GENERATED_ASSETS_CAR}")
```

Comment:
this test will only run at configure time when the file hasn't been generated yet.